### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by rodinux**
## Problem

- *api and webdav connexions not working and possibility to change smtp mail sender*

## Solution

- *remove ssowat restriction to match api, add a config panel*

## Also

- enable thumbnail generation (for pdf, images, 
- enable pdf indexing
- various docs enhancements 
- apply appropriate (more secure) permissions on install_dir & data_dir
- enable NTP to ensure OTP/2FA is working properly

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by rodinux** [comment]
!testme

**Comment by rodinux** [comment]
@OniriCorpe, est-ce que le webdav fonctionne pour toi ainsi ?

**Comment by OniriCorpe** [comment]
[`24de38b` (#69)](https://github.com/YunoHost-Apps/paheko_ynh/pull/69/commits/24de38b10961ba4f3f0fcb4ac79971b0784aa906) fixes this:

```text
Info : [#####++.............] > Updating a configuration file...
Attention : File /var/www/paheko/config.local.php has been manually modified since the installation or last upgrade. So it has been duplicated in /var/cache/yunohost/appconfbackup//var/www/paheko/config.local.php.backup.20240904.030327
Attention : --- /var/cache/yunohost/appconfbackup//var/www/paheko/config.local.php.backup.20240904.030327       2024-09-04 02:59:27.504738543 +0200
Attention : +++ /var/www/paheko/config.local.php        2024-09-04 03:03:29.037096282 +0200
Attention : @@ -990,4 +990,4 @@
Attention :   * Chemin vers le fichier pour des configurations personnelles qui ne sera pas écraser
Attention :   * lors des mises à jour. Cette ligne sera décommentée dès la première mise à jour.
Attention :   */
Attention : -require '/home/yunohost.app/paheko/data/config.local.user.php';
Attention : +//require '/home/yunohost.app/paheko/data/config.local.user.php';
Info : [####################] > Upgrade of paheko completed
```

**Comment by OniriCorpe** [comment]
i enabled thumbnail generation ^w^

screenshot of paheko documents with thumbnails

**Comment by OniriCorpe** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -60,4 +60,5 @@ ram.runtime = "50M"
     webdav.auth_header = false
 
     [resources.apt]
-    packages = "sqlite3, php8.2-sqlite3, php8.2-gd, php8.2-intl, php8.2-cli, php8.2-gnupg, php8.2-mbstring, php8.2-zip"
+    packages = "sqlite3, php8.2-sqlite3, php8.2-imagick, php8.2-intl, php8.2-cli, php8.2-gnupg, php8.2-mbstring, php8.2-zip, mupdf-tools, ffmpeg"
```
Comment (OniriCorpe) [inline manifest.toml:63]: bt i replaced gd with imagick as the paheko docs says they prefer imagick: 

> Images : nécessite d'avoir l'extension __PHP Imagick (de préférence)__ ou GD.

https://fossil.kd2.org/paheko/wiki?name=Configuration/Thumbnails

**Comment by OniriCorpe** [comment]
!testme

**Comment by rodinux** [comment]
!testme

File: manifest.toml [hunk 2]
```diff
@@ -47,6 +47,11 @@ ram.runtime = "200M"
     # note: permissions are set in the install/upgrade/restore script
 
     [resources.data_dir]
+    dir = "/home/yunohost.app/__APP__"
```
Comment (OniriCorpe) [inline manifest.toml:50]: beware, i already set the correct permissions in the install/upgrade/restore scripts

**Comment by OniriCorpe** [comment]
i think our work is complete for now

maybe ask for some user feedback on the forum before merging but that's it

right?

**Comment by rodinux** [comment]
Nice thanks a lot @OniriCorpe ... Then I think I have to clean the code... If you are fine with the permissions to use and think something is wrong tell me... For exemple I don't record why I keep the owner of `config.local.php` with $app:$app, also for `config.local.user.php`, but there was a reason... 

I fixed the install with  $data_dir directory , so don't need more the hack with the last line `require '__DATA_DIR__/data/config.local.user.php';`  of the `config.local.php` commented and decommented on each update XP

**Comment by OniriCorpe** [comment]
a final CI test :D 

!testme

**Comment by rodinux** [comment]
do we merge this PR @OniriCorpe ?

**Comment by OniriCorpe** [comment]
hmm yes, everything is fine on my side

**Review by OniriCorpe — APPROVED** [review]

**Pull request merged**